ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobStoreHardDeleteTest.java

```diff
@@ -138,12 +138,15 @@
     private MessageFormatInputStream getPutMessage(StoreKey key, BlobProperties blobProperties, byte[] usermetadata,
         byte[] blob, int blobSize, short blobVersion, BlobType blobType)
         throws MessageFormatException {
-      if (blobVersion == MessageFormatRecord.Blob_Version_V1) {
-        return new PutMessageFormatInputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
-      } else {
-        return new PutMessageFormatBlobV2InputStream(key, blobProperties, ByteBuffer.wrap(usermetadata),
-            new ByteBufferInputStream(ByteBuffer.wrap(blob)), blobSize, blobType);
+      switch (blobVersion) {
```

Comment:
You could make this consistent with the rest of the tests. `switch-case` preferably, but up to you.